src/widgets/ActivityGroups/model/hooks/useIntegrationsSync.ts

```diff
@@ -12,13 +14,15 @@ export const useIntegrationsSync = ({ appletDetails }: Props) => {
   const dispatch = useAppDispatch()
   const rootState = useAppSelector(state => state)
 
-  const appletIntegrationService = new AppletIntegrationsService(rootState, dispatch)
-
   const isLorisIntegrationAvailable = appletDetails.integrations.some(integration =>
     AVAILABLE_INTEGRATIONS.includes(integration),
   )
 
-  if (isLorisIntegrationAvailable) {
-    appletIntegrationService.applyIntegrations(appletDetails)
-  }
+  useEffect(() => {
+    const appletIntegrationService = new AppletIntegrationsService(rootState, dispatch)
+
+    if (isLorisIntegrationAvailable) {
```

Comment:
Please leave out if block and isLorisIntegrationAvailable variable

Reply:
Done